### PR TITLE
INT-1752 UI for GH issue creation

### PR DESCRIPTION
**Pull request by hbjORbj**
- SourceControlPanelProvider
- CreateIssue view

claap: https://app.claap.io/intuita/int-1752-ui-for-gh-issue-creation-c-BT2DRbCjzO-mOjR_uzh5Ntg

### CODE REVIEW

File: intuita-webview/src/sourceControl/CreateIssue/index.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import styles from './style.module.css';
+import { IssueFormData } from '../../../../src/components/webview/webviewEvents';
+
+type Props = Readonly<{
+	title: string;
+	body: string;
+}>;
+
+const CreateIssue = ({ title, body }: Props) => {
+	const [loading] = useState(false);
```
Comment (DmytroHryshyn) [inline intuita-webview/src/sourceControl/CreateIssue/index.tsx:17]: why we dont have state setter here?

File: src/persistedState/codecs.ts [hunk 2]
```diff
@@ -112,6 +112,29 @@ export const persistedStateCodecNew = buildTypeCodec({
...
+				kind: t.literal('ISSUE_CREATION'),
+				jobHash: t.string,
+				oldFileContent: t.string,
+				newFileContent: t.string,
+			}),
+			buildTypeCodec({
+				kind: t.literal('IDLENESS'),
+			}),
+		]),
+		// buildTypeCodec({
```
Comment (DmytroHryshyn) [inline src/persistedState/codecs.ts:127]: Do we need this commented code?

File: intuita-webview/src/sourceControl/CreateIssue/index.tsx (hunk 1 above)
Comment (grzpab) [reply intuita-webview/src/sourceControl/CreateIssue/index.tsx:17]: b/c this functionality is not yet connected to the backend

File: src/persistedState/codecs.ts (hunk 2 above)
Comment (grzpab) [reply src/persistedState/codecs.ts]: nope, fixed

**Review by DmytroHryshyn — APPROVED** [review]

**Pull request merged**